integration-test/build.gradle.kts

```diff
@@ -132,6 +131,21 @@ dependencies {
   testImplementation(libs.trino.jdbc)
 
   testRuntimeOnly(libs.junit.jupiter.engine)
+
```

Comment:
spark connector is planning to move spark it from `integrate test` to `spark-connector` module to avoid potential class path conflict and support multi spark version #3307 , maybe flink could place it in flink connector at first time.

Reply:
yes, the implement will more like hudi not iceberg. https://docs.google.com/document/d/1t7a8Obe8JIuQzhL_USHFJlPLO_5ODsqOQPA6-jNUMSw/edit